aws-rds-dbinstance/src/test/java/software/amazon/rds/dbinstance/UpdateHandlerTest.java

```diff
@@ -1641,4 +1648,80 @@ public void handleRequest_ObserveFailureEvent() {
         verify(rdsProxy.client(), times(1)).describeEvents(any(DescribeEventsRequest.class));
         verify(rdsProxy.client(), times(2)).describeDBInstances(any(DescribeDbInstancesRequest.class));
     }
+
+    @Test
+    public void handleRequest_startAutomaticBackupReplication() {
+        final CallbackContext context = new CallbackContext();
+        context.setCreated(true);
+        context.setUpdated(true);
+        context.setRebooted(true);
+        context.setUpdatedRoles(true);
+        context.setAddTagsComplete(true);
+        context.setAutomaticBackupReplicationStarted(false);
+        context.setAutomaticBackupReplicationStopped(true);
+        context.getProbingContext().setProbingEnabled(false);
+
+        proxy = Mockito.spy(proxy);
+
+        final RdsClient crossRegionRdsClient = mock(RdsClient.class);
+        final ProxyClient<RdsClient> crossRegionRdsProxy = mockProxy(proxy, crossRegionRdsClient);
+        doReturn(crossRegionRdsProxy).when(proxy).newProxy(ArgumentMatchers.<Supplier<RdsClient>>any());
+
+        test_handleRequest_base(
+                context,
+                () -> DB_INSTANCE_ACTIVE.toBuilder()
+                        .dbInstanceAutomatedBackupsReplications(Collections.singletonList(DBInstanceAutomatedBackupsReplication.builder()
+                                .dbInstanceAutomatedBackupsArn(AUTOMATIC_BACKUP_REPLICATION_ARN_ALTER).build()))
+                        .build(),
+                () -> RESOURCE_MODEL_BLDR()
+                        .automaticBackupReplicationRegion(AUTOMATIC_BACKUP_REPLICATION_REGION)
+                        .build(),
+                () -> RESOURCE_MODEL_BLDR()
+                        .automaticBackupReplicationRegion(AUTOMATIC_BACKUP_REPLICATION_REGION_ALTER)
+                        .build(),
+                expectSuccess()
+        );
+
+        verify(crossRegionRdsProxy.client(), times(1)).startDBInstanceAutomatedBackupsReplication(any(StartDbInstanceAutomatedBackupsReplicationRequest.class));
```

Comment:
Why `verifyAccessPermissions(rdsClient);` did not catch the missing permission in resource schema `StartDBInstanceAutomatedBackupsReplication, and StopDBInstanceAutomatedBackupsReplication.`?